app/utils/agendaitem-utils.js

```diff
@@ -70,7 +70,7 @@ export const setModifiedOnAgendaOfAgendaitem = async(agendaitem) => {
  * @param resetFormallyOk
  * @returns {Promise<void>}
  */
-export const saveChanges = async(agendaitemOrSubcase, propertiesToSetOnAgendaitem, propertiesToSetOnSubcase, resetFormallyOk) => {
+export const saveChanges = async(agendaitemOrSubcase, propertiesToSetOnAgendaitem, propertiesToSetOnSubcase, resetFormallyOk, store) => {
```

Comment:
I don't love this change, it feels weird to pass in the store to this util so that it can fetch the needed agendaitems.

An alternative is fetching the agendaitems through the relations on the subcase, i.e. `subcase.agendaActivities.mapBy('agendaitems')`, getting their agenda & checking status, but that implies a bunch of extra queries.

Alternatively this function could be split into two, one for saving changes "from" a subcase, and other one for saving "from" an agendaitem, and then it's the caller's responsibility to fetch the necessary agendaitems, but that means some code duplication in ~5 places.